bundles/model/org.eclipse.smarthome.model.notification.ui/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding/<project>=UTF-8
```

Comment:
Please remove this file because it causes the default Eclipse project settings to be overridden. See also #6106.